packages/core/src/Modules/Callback/CallbackPage.tsx

```diff
@@ -13,6 +13,7 @@ const CallbackPage = () => {
                 localStorage.setItem('config.tokens', JSON.stringify(tokens));
                 localStorage.setItem('config.account1', tokens.token1);
                 localStorage.setItem('active_loginid', tokens.acct1);
+                sessionStorage.setItem('active_loginid', tokens.acct1);
```

Comment:
Since after user logs out, there is no `tradershub_redirect_to` session storage, thus the if condition `if (redirectTo)` will never execute and `active_loginid` session storage is null. We just set a default value here

<img width="485" alt="Screenshot 2025-03-13 at 9 38 13 AM" src="https://github.com/user-attachments/assets/643a7f9c-b229-44bc-b53d-804f16bafcc6" />

Reply:
https://github.com/user-attachments/assets/e392447a-0740-40c3-b4a7-cf51d3533219